dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -63,28 +70,63 @@ class SentryStackTraceFactory {
-
-    if (frame.line != null && frame.line >= 0) {
-      sentryStackFrame = sentryStackFrame.copyWith(lineNo: frame.line);
-    }
+  SentryStackFrame encodeStackTraceFrame(Frame frame,
+      {bool symbolicated = true}) {
+    final member = frame.member;
+
+    SentryStackFrame sentryStackFrame;
+
+    if (symbolicated) {
+      final fileName = frame.uri.pathSegments.isNotEmpty
+          ? frame.uri.pathSegments.last
+          : null;
+
+      final abs = '$eventOrigin${_absolutePathForCrashReport(frame)}';
+
+      sentryStackFrame = SentryStackFrame(
+        absPath: abs,
+        function: member,
+        // https://docs.sentry.io/development/sdk-dev/features/#in-app-frames
+        inApp: isInApp(frame),
+        fileName: fileName,
+        package: frame.package,
+      );
+
+      if (frame.line != null && frame.line >= 0) {
+        sentryStackFrame = sentryStackFrame.copyWith(lineNo: frame.line);
+      }
 
-    if (frame.column != null && frame.column >= 0) {
-      sentryStackFrame = sentryStackFrame.copyWith(colNo: frame.column);
+      if (frame.column != null && frame.column >= 0) {
+        sentryStackFrame = sentryStackFrame.copyWith(colNo: frame.column);
+      }
+    } else {
+      // if --split-debug-info is enabled, thats what we see:
+      // warning:  This VM has been configured to produce stack traces that violate the Dart standard.
+      // ***       *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
+      // unparsed  pid: 30930, tid: 30990, name 1.ui
+      // unparsed  build_id: '5346e01103ffeed44e97094ff7bfcc19'
+      // unparsed  isolate_dso_base: 723d447000, vm_dso_base: 723d447000
+      // unparsed  isolate_instructions: 723d452000, vm_instructions: 723d449000
+      // unparsed      #00 abs 000000723d6346d7 virt 00000000001ed6d7 _kDartIsolateSnapshotInstructions+0x1e26d7
+      // unparsed      #01 abs 000000723d637527 virt 00000000001f0527 _kDartIsolateSnapshotInstructions+0x1e5527
+      // unparsed      #02 abs 000000723d4a41a7 virt 000000000005d1a7 _kDartIsolateSnapshotInstructions+0x521a7
+      // unparsed      #03 abs 000000723d624663 virt 00000000001dd663 _kDartIsolateSnapshotInstructions+0x1d2663
+      // unparsed      #04 abs 000000723d4b8c3b virt 0000000000071c3b _kDartIsolateSnapshotInstructions+0x66c3b
+
+      // we are only interested on the #01, 02... which contains the 'abs' addresses.
+      if (member.contains('abs') && member.contains('virt')) {
+        // TODO: use proper Regex, this works for now
+        final indexAbs = member.indexOf('abs');
+        final indexVirt = member.indexOf('virt');
+        final instructionAddr =
+            '0x${member.substring(indexAbs + 4, indexVirt - 1)}';
+
+        sentryStackFrame = SentryStackFrame(
+          instructionAddr: instructionAddr,
+          platform: 'native', // to trigger symbolication
+          symbolicated: false, // signal to load image list from Native SDKs
```

Comment:
could we use the `platform=native` to trigger loading images instead?

Reply:
I thought about it but iOS events are `platform=native` (Native) and they should not load, at least for now, but no strong feelings, wdty? really your call